WordPress/Classes/ViewRelated/Me/MeViewController.swift

```diff
@@ -414,6 +414,26 @@ class MeViewController: UITableViewController, UIViewControllerRestoration {
         headerView.onGravatarPress = { [weak self] in
             self?.presentGravatarPicker()
         }
+        headerView.onDroppedImage = { [weak self] image in
+            let imageCropViewController = ImageCropViewController.init(image: image!)
+            imageCropViewController.maskShape = .square
+            imageCropViewController.shouldShowCancelButton = true
+
+            imageCropViewController.onCancel = { () -> Void in
+                self?.dismiss(animated: true, completion: nil)
+                self?.gravatarUploadInProgress = false
+            }
+            imageCropViewController.onCompletion = { (UIImage, Bool) -> Void in
+                self?.dismiss(animated: true, completion: nil)
+                if let updatedGravatarImage = image {
```

Comment:
Is `image` here supposed to be the `image` that was passed to the `onDroppedImage` block or is it supposed to be the `UIImage` passed to the `onCompletion` block? I'm guessing the latter but could be mistaken.  If so, maybe rename `UIImage` and reference it instead?

Reply:
Right, I think it should be the image passed into this block that's used (the one that's been cropped / resized), rather than the image in the enclosing block. The block definition should used names for the parameters, not their types:

`imageCropViewController.onCompletion = { [weak self] image, _ in`

(we're not using the bool parameter, so that can be omitted)